src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -267,13 +267,20 @@ <H2>Address Cache</H2>
 	policy, while a value of 0 (zero) means no caching. The default value
 	is -1 (forever) if a security manager is installed, and implementation-specific
 	when no security manager is installed.</P>
+	<LI><P><B>{@systemProperty networkaddress.cache.stale.ttl}</B> (default: see below)<BR>
+	Value is an integer corresponding to the number of seconds the stale names
+	will be kept in the cache. The name is considered stale if it expired and
+	attempts to refresh it were unsuccessful. This property is useful if it is
+	preferable to use a stale name rather than result of unsuccessful name lookup.
+	A value of 0 (zero) means do not use stale names.
+	The default value is implementation-specific.</P>
```

Comment:
```suggestion
	A value of 0 (zero) or if the property is not set means do not use stale names.
	By default, the property is not set.</P>
```

Reply:
We probably should specify what happens if the stale property is negative. Looks like it might be ignored, which I guess is okay.

Reply:
I have update the text about "property is not set" case, but leave the text about "default value" as implementation-specific, similarly to what we have in the text about "networkaddress.cache.ttl" property, and this allow to change this value in some situations.

Reply:
Right now the negative value is ignored, and is not mentioned in the new spec, But it could be used to cover "always use the stale data on error" case. So we can change this in the future, but probably it will not be a big deal to update the spec if that will happen.

Reply:
I believe we do need to specify what happens if the value is negative. We cannot leave that as "undefined behaviour that may change in the future".

Reply:
will update the spec.